src/RichText.js

```diff
@@ -0,0 +1,82 @@
+import {Editor, ContentState, convertFromHTML, DefaultDraftBlockRenderMap, DefaultDraftInlineStyle, CompositeDecorator} from 'draft-js';
+const ReactDOM = require('react-dom');
+var sdk = require('./index');
+
+const BLOCK_RENDER_MAP = DefaultDraftBlockRenderMap.set('unstyled', {
+    element: 'p' // draft uses <div> by default which we don't really like
```

Comment:
Actually, even <p> probably won't work. @ara4n says we should use no enclosing tags and <br> for line breaks. While I haven't tested this I'm fairly sure it messes up HTMLtoContentState in a way that content that's reloaded from history doesn't correspond with what you actually entered.